src/core/routes/routes.tsx

```diff
@@ -126,13 +126,19 @@ export const routes: Route[] = [
         },
         element: () => import('../../pages/feed/refactored/feedDetails/feedDetails').then((m) => <m.default />),
       },
+    ],
+  },
+  {
+    path: 'jobs/',
+    element: <Layout />,
```

Comment:
I realized that we need to add this line here :  loader: jobsPageLoader, 
to avoid logout issue on page refresh
this is a temp solution based on current implementation and should be revised on routing refactoring